src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -2027,6 +2027,9 @@ public static class Warning
 
       public static string LargeNumberOfOutputPoints(int numberOfPoints) =>
          $"The selected output resolution will generate {numberOfPoints} points and may severely impact the software performance.\nAre you sure you want to run with these setting? If not, consider changing output resolution in simulations settings";
+
+      public static string GlobalParameterIsDefinedInMultipleSpatialStructures(string parameterName) =>
```

Comment:
I think having the same parameter defined in two different spatial structure is a bit weird. Alternatively we could just take the Global Molecule parameter from the LAST spatial structure. Thoughts @PavelBal @Yuri05 @rwmcintosh

Reply:
Actually I still don't quite understand how it comes to this situation.
I thought that if we have >1 spatial structure with the global MoleculeProperties container - than the global MoleculeProperties container from the **last** spatial structure would completely **replace** the one from the previous spatial structure(s)?

Reply:
>I feel that we should just use the last global container 

I would say: yes, this sounds more reasonable for me.

Reply:
see new implementation